framework/src/postprocessors/ElementExtremeValue.C

```diff
@@ -44,7 +45,6 @@ ElementExtremeValue::validParams()
 ElementExtremeValue::ElementExtremeValue(const InputParameters & parameters)
   : ElementVariablePostprocessor(parameters),
     _type((ExtremeType)(int)parameters.get<MooseEnum>("value_type")),
-    _value(_type == 0 ? -std::numeric_limits<Real>::max() : std::numeric_limits<Real>::max()),
```

Comment:
I'm going to test this with valgrind - if anyone uses this value before initialize() for some reason, we may get an uninitialized vailure